agent/php_user_instrument.h

```diff
@@ -82,6 +106,31 @@ typedef struct _nruserfn_t {
 
 extern nruserfn_t* nr_wrapped_user_functions; /* a singly linked list */
 
+/*
+ * Purpose : Determine if a func matches a wraprec.
+ *
+ * Params  : 1. The wraprec to match to a zend function
+ *           2. The zend function to match to a wraprec
+ *
+ * Returns : True if the class/function of a wraprec match the class function
+ *           of a zend function.
+ */
+extern bool nr_php_wraprec_matches(nruserfn_t* p, zend_function* func);
```

Comment:
can you please make this static inline?
We aren't using it externally anymore, it can just exist in the .c file to hopefully add a little more speed.

Reply:
@ZNeumann I made suggestion not to export this function in https://github.com/ZNeumann/newrelic-php-agent/pull/1/commits/6eacc32388de762c301b35b1bab72d185afe07ba so keep it in mind if you plan to accept https://github.com/ZNeumann/newrelic-php-agent/pull/1 before applying `inline` keyword.

Reply:
Made inline in [93baea2](https://github.com/newrelic/newrelic-php-agent/pull/568/commits/93baea2610bc2b2f373a0a04a36a27f31313c62b)